.github/workflows/webhook.yml

```diff
@@ -0,0 +1,17 @@
+name: Discord Automation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  commit-stream:
+    runs-on: ubuntu-latest
+    steps:
+    - env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+      uses: Ilshidur/action-discord@0.3.2
+      if: "${{ env.DISCORD_WEBHOOK != '' }}"
```

Comment:
Ideally the _workflow_ should not run, rather than just skipping a step in a single job. Is that possible?

Reply:
Yeah, I looked and didn't find any way to stop the "workflow" itself from running, since I'm pretty sure it *has* to run in order for the secret to be loaded into an environment variable in order to be used in a conditional. 
Though if anyone does know of a way, I'd be more than happy to add it.